airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterWorkflow.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.temporal.scheduling;
+
+import io.temporal.workflow.QueryMethod;
+import io.temporal.workflow.SignalMethod;
+import io.temporal.workflow.WorkflowInterface;
+import io.temporal.workflow.WorkflowMethod;
+
+@WorkflowInterface
+public interface ConnectionUpdaterWorkflow {
```

Comment:
A few comments would be useful here to explain the purpose of each of these.

Reply:
I am adding a note about doing it when implementing the method.

Reply:
Added